docs/metecho-how-to.md

```diff
@@ -0,0 +1,150 @@
+# Steps to Guide a Team Through Metecho
```

Comment:
Could we add an intro paragraph on what Metecho does and why its useful? That might encourage people to learn and use it. Something highlighting that usually contributing to SF projects requires editing code on GitHub but Metecho makes it much easier and interactive.

Reply:
That's an excellent idea